package/MDAnalysis/analysis/density.py

```diff
@@ -306,7 +306,7 @@ def make_density(self):
             warnings.warn(msg)
             return
 
-        dedges = map(np.diff, self.edges)
+        dedges = list(map(np.diff, self.edges))
```

Comment:
This is a complicated way for a list comprehension `[np.diff(e) for e in self.edges]`